### PR TITLE
Update GUI

### PR DESCRIPTION
**Pull request by ph-nathan**
#48 
- Split GUI into two columns
- Modify color scheme of application
- Change logo 
- Add Event related card and list elements 

image

### CODE REVIEW

File: src/main/java/seedu/address/ui/EventCard.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.event.Event;
+
+/**
+ * A UI component that displays information of a {@code Member}.
```
Comment (andrechuakj) [inline src/main/java/seedu/address/ui/EventCard.java]: Should be Event instead of Member

**Review by SelwynAng — APPROVED** [review]
LGTM!

**Review by andrechuakj — COMMENTED** [review]
Just a nit, LGTM

File: src/main/java/seedu/address/ui/EventCard.java (hunk 1 above)
Comment (wr1159) [inline src/main/java/seedu/address/ui/EventCard.java]:
```suggestion
 * A UI component that displays information of a {@code Event}.
```

**Pull request merged**